src/main/scala/forms/DateWithDaysField.scala

```diff
@@ -4,17 +4,18 @@ import controllers.JsonHelpers
 import forms.validation.{DateWithDaysValidator, DateWithDaysValues, FieldError, FieldHint}
 import models.{ApplicationFormSection, ApplicationOverview, Question}
 import org.joda.time.format._
-import play.api.Logger
 import play.api.libs.json.JsObject
 import play.twirl.api.Html
 
-case class DateWithDaysField(name: String, validator: DateWithDaysValidator) extends Field {
+case class DateWithDaysField(name: String, allowPast: Boolean, minValue: Int, maxValue: Int) extends Field {
```

Comment:
The Validator was here before, but this pattern keeps spreading -  why Option is not used here?